src/server/visual-diff-plugin.js

```diff
@@ -56,6 +59,31 @@ async function clearDiffPaths(dir) {
 	}
 }
 
+async function createComparisonPNGs(original, newSize) {
+	const resizedPNGs = [];
+	[
+		{ name: 'top', coord: 0 },
+		{ name: 'center', coord: Math.floor((newSize.height - original.height) / 2) },
+		{ name: 'bottom', coord: newSize.height - original.height }
+	].forEach(y => {
+		[
+			{ name: 'left', coord: 0 },
+			{ name: 'center', coord: Math.floor((newSize.width - original.width) / 2) },
+			{ name: 'right', coord: newSize.width - original.width }
+		].forEach(x => { // TODO: position added for reports, remove/adjust as needed
+			if (original.width === newSize.width && original.height === newSize.height) {
+				resizedPNGs.push({ png: original, position: `${y.name}-${x.name}` });
```

Comment:
The position can be adjusted to whatever if we want to use it for the report, I don't actually use it anywhere.  I sued it for testing, and because it made these nested `for`s a lot easier to read.

Reply:
I don't _think_ the reports are going to need it, since they're going to use the resized images anyway which will always both be the same size.

Reply:
Yeah it was only if we wanted to call it out for extra clarity.  I could remove this and switch to just comments for each object, like
```
	[
		0, // top
		Math.floor((newSize.height - original.height) / 2), // center
		newSize.height - original.height // bottom
	].forEach(y => {
		[
			0, // left
			Math.floor((newSize.width - original.width) / 2), // center
			newSize.width - original.width // right
		].forEach(x => {
```
?

Reply:
I guess keep it for now and I'll see if it can be used in a useful way. Your TODO can remind us to remove it later if not!